athletics.lic

```diff
@@ -0,0 +1,170 @@
+custom_require.call(%w(common common-travel))
+
+class Athletics
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    @settings = get_settings()
+    @athletics_options = get_data('athletics').athletics_options
+
+    @climbing_target = get_data('athletics').practice_options[@settings.climbing_target]
+    
+    @start_skill=DRSkill.getxp('Athletics')
+    @end_skill=@start_skill+15
+    if(@end_skill>29)
+      @end_skill=29
+    end
+
+    if @climbing_target
+      walk_to(@climbing_target['id'])
+      climb_practice(@climbing_target['target'], @climbing_target['hide'])
+    elsif @settings.hometown == 'Crossing' || (@settings.hometown == 'Riverhaven' and DRSkill.getrank('Athletics') > 140)
```

Comment:
Something like this? Or should I wrap all of this in a riverhaven_athletics method?